pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/LinuxBrokerHostUsageImpl.java

```diff
@@ -105,9 +105,14 @@ public void calculateBrokerHostUsage() {
         double totalNicUsageRx = getTotalNicUsageRxKb(nics);
         double totalCpuLimit = getTotalCpuLimit();
 
-        SystemResourceUsage usage = new SystemResourceUsage();
         long now = System.currentTimeMillis();
         double elapsedSeconds = (now - lastCollection) / 1000d;
+        if (elapsedSeconds <= 0) {
+            log.warn("elapsedSeconds {} is not expected,skip this round of calculateBrokerHostUsage", elapsedSeconds);
```

Comment:
suggest add a space after `,`

Reply:
OK

Reply:
Please review again, thank you! Following your suggestion, I made the following changes: @Shoothzj

add a space in log content :
log.warn("elapsedSeconds {} is not expected, skip this round of calculateBrokerHostUsage", elapsedSeconds);